packages/bpk-component-calendar/src/BpkCalendarGrid.js

```diff
@@ -215,9 +215,9 @@ BpkCalendarGrid.defaultProps = {
   cellClassName: null,
 };
 
-const BpkCalendarGridWithTransition = addCalendarGridTransition(
-  BpkCalendarGrid,
-);
+const BpkCalendarGridWithTransition = isDeviceIos()
```

Comment:
We should probably add a comment here to explain why we're doing this, as it's not immediately obvious. 

```suggestion
// On iOS, having transitions causes accessibility issues, so we disable it (KOA-4467).
const BpkCalendarGridWithTransition = isDeviceIos()
```

Reply:
Sounds good to me!